android/src/main/java/com/taskrabbit/zendesk/RNZendeskChatModule.java

```diff
@@ -376,5 +378,70 @@ public void update(ChatState chatState) {
                 }
             }
         });
-    }
+	}
+
+	@ReactMethod
+	public void sendMessage(String message, Promise promise) {
+		ChatLog log = Chat.INSTANCE.providers().chatProvider().sendMessage(message);
+
+		DeliveryStatus status = log.getStatus();
+
+		if (status == DeliveryStatus.DELIVERED) {
+			// not sure if this is the messageId like in the Objective-C API
+			promise.resolve(log.getId());
+		} else {
+			promise.reject();
+		}
+
+	}
+
+	@ReactMethod
+	public void sendRating(String rating, Promise promise) {
+    	zendesk.chat.ChatRating parsedRating = zendesk.chat.ChatRating.valueOf(rating);
+
+		Chat.INSTANCE.providers().chatProvider().sendChatRating(parsedRating, null);
```

Comment:
I don't know how to set a callback there. I tried with a lambda function, but I don't know how to get the signature of the parameters received in the lambda function... so if thought it was better to avoid it, but I'm returning a promise to have the same iOS API